### ISSUE TITLE
F-Droid Compatibility

### ISSUE THREAD

**Issue by meypod** [maplibre/maplibre-react-native#433]
Hi

Since Maplibre is using `https://dl.bintray.com/maplibre/maplibre-gl-native` maven repo, f-droid builds are failing. I think the library should be hosted on a well-known maven repo.

related: https://gitlab.com/fdroid/fdroiddata/-/merge_requests/15592

thank you for your hard work

**Comment by tyrauber** [comment]
I don't even see [the Android documentation](https://maplibre.org/maplibre-native/docs/book/android/getting-started-guide.html) adding this line?! A search for [`bintray.com` comes up empty](https://github.com/search?q=repo%3Amaplibre%2Fmaplibre-native%20bintray.com&type=code).  I wonder if this is a legacy requirement of older versions of the SDK.  Perhaps the newer SDK doesn't require it and just removing this line will work?

**Comment by meypod** [comment]
hi
it's because it's here, in maplibre-react-native:
https://github.com/maplibre/maplibre-react-native/blob/9fcf6d162b49044a3789eadc5c1d4b784444728d/android/rctmln/build.gradle#L56

**Comment by meypod** [comment]
@tyrauber I seem to be able to build it locally with the maven block removed, so I guess it's a legacy requirement of maplibre-react-native ?

**Comment by meypod** [comment]
@tyrauber while the build succeeds, Map no longer works when building without the maven.

**Comment by tyrauber** [comment]
No worries! The native SDKs have been updated significantly since the 9.0 SDKs. My guess it is probably can be removed, I just haven't got to testing it myself.

**Comment by birkskyum** [comment]
The latest MapLibre Native core is AFAIK compatible with F-Droid:
- https://github.com/maplibre/maplibre-native/issues/379

**Comment by meypod** [comment]
As far as i remember you just need to remove the maven block mentioned and it's good to go

**Comment by KiwiKilian** [comment]
F-Droid should be usable with the latest v10 prereleases. Can anyone confirm this? How could I check this myself?

Also, which part would be flagged by F-Droid for using Google? Could we use a variable in the `build.gradle` to optionally add the `implementation "com.google.android.gms:play-services-location:21.3.0"` line? Or would also the sheer existence of the implementation, which would be conditionally choosen be a problem?

Basically, I would like to have this [patch](https://github.com/maplibre/maplibre-react-native/issues/446#issuecomment-2375235171) in the package, toggled by a variable or something, allowing this functionality to be toggle by an Expo plugin variable.

**Comment by KiwiKilian** [comment]
Closing as this has been released with v10. Please add more info, if there is still a problem with F-Droid compatibility.
